tutorials/core-concepts/pieces/README.md

```diff
@@ -0,0 +1,17 @@
+---
+title: "Reusable Content With Pieces"
+layout: reference
+children:
+  - reusable-content-with-pieces
+  - displaying-pieces-with-widgets
+  - browsing-directory-of-pieces
+  - joins
+---
+
+When you build a website, you could look at it like a puzzle. Each element of the site needs to fit together so you can see the whole picture. In Apostrophe, any kind of structured content is a “piece” and that content becomes pieces of the puzzle that you’re putting together as you build your site. You can use pieces for things ranging from defining a standard format for a blog post to creating a new entity type like “bikes” for a bike shop. Apostrophe uses pieces internally to define different content out of the box – and many other things like users, that you might not think of as “content” – and you can harness that same power to put the pieces of your own puzzle together.
+
+Take a bike shop as an example: you could create individual pages for everything you sell – each type of bike, each type of helmet, and each repair service that you offer – but you’d up with a messy site that would quickly become unmanageable. So instead, you take a step back, and think about the components that you need to build your content, and create a piece type for each one: bikes, helmets, accessories, services, etc.. Each individual item is created as a piece. When you get a new model bike in stock, you create a new piece from the bike piece type, and the associated page and entry is automatically created. In Apostrophe, pieces are the key to designing, building, and maintaining your site.
```

Comment:
This makes it sound like pieces are better than pages in some way.

Pages and pieces have distinct use cases. If it lives in only one place on the site and has a parent/child relationship to other pages, it's a page. If it might appear in many places on the site according to your needs, it's a piece.

As for this: "When you get a new model bike in stock, you create a new piece from the bike piece type, and the associated page and entry is automatically created"

An Apostrophe page in the tree is not created, but I know what you mean. So let's try to nail down the difference between a page in the tree, and a permalink for a piece.

I would suggest: "When you get a new model bike in stock, you create a new piece via "Bikes" on the admin bar. Then, if you wish, you can display it anywhere on the site via a widget, and also provide a way to browse all bikes and reach a detail page or "permalink" page for each one with no extra effort via Apostrophe's 'pieces-pages" feature."